validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorLoader.java

```diff
@@ -48,11 +48,12 @@
 
     for (int i = 0; i < keypairs.size(); i++) {
       BLSKeyPair keypair = keypairs.get(i);
-      int port = Constants.VALIDATOR_CLIENT_PORT_BASE + i;
+      int port =
+          Constants.VALIDATOR_CLIENT_PORT_BASE + i + keyProvider.getValidatorPortStartIndex();
       new ValidatorClient(keypair, port);
       ManagedChannel channel =
           ManagedChannelBuilder.forAddress("localhost", port).usePlaintext().build();
-      STDOUT.log(Level.DEBUG, "i = " + i + ": " + keypair.getPublicKey().toString());
+      STDOUT.log(Level.INFO, "i = " + i + ": " + keypair.getPublicKey().toString());
```

Comment:
if you pass in -l=DEBUG into the run.sh script this printout will appear.  We should probably keep it as debug by default